homeassistant/components/iskra/sensor.py

```diff
@@ -163,6 +167,44 @@ class IskraSensorEntityDescription(SensorEntityDescription):
 )
 
 
+def get_counter_entity_description(
+    counter: Counter,
+    index: int,
+    entity_name: str,
+) -> IskraSensorEntityDescription:
+    """Dynamically create IskraSensor object as energy meter's counters are customizable."""
+
+    key = entity_name.format(index + 1)
+
+    if entity_name == ATTR_NON_RESETTABLE_COUNTER:
+        entity_description = IskraSensorEntityDescription(
+            key=key,
+            translation_key=key,
+            state_class=SensorStateClass.TOTAL_INCREASING,
+            value_func=(lambda device: device.counters.non_resettable[index].value),
```

Comment:
you only have to parenthesize lambdas if they span more than one line

Reply:
will remove